src/runtime_src/core/common/api/command.h

```diff
@@ -64,6 +64,12 @@ class command : public std::enable_shared_from_this<command>
   virtual void
   notify(ert_cmd_state) const = 0;
 
+  /**
+  * get_hw_context() - get hw context of command buffer
+  */
```

Comment:
```suggestion
  /**
  ```suggestion
  * get_hwctx_handle() - get submission hw context of command buffer
  *
  * The submission hw context is the hardware context used for command execution.
  * This context is used in multi-xclbin / slot support when submitting a command with 
  * execbuf when the core implementation does not support hardware queue.
```
  */
```

Reply:
fixed